domains/authorization/Authorization.API/API/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -50,15 +43,19 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .ValueGeneratedOnAdd()
                         .HasColumnType("uuid");
 
+                    b.Property<int>("ClientType")
+                        .HasColumnType("integer");
+
                     b.Property<Guid>("IdpClientId")
                         .HasColumnType("uuid");
 
                     b.Property<string>("Name")
                         .IsRequired()
                         .HasColumnType("text");
 
-                    b.Property<int>("Role")
-                        .HasColumnType("integer");
+                    b.Property<string>("RedirectUrl")
+                        .IsRequired()
+                        .HasColumnType("text");
```

Comment:
Why store as text? Shouldn't it be stored as enum(int)?

Reply:
This is the redirectUrl, it is a string